### ISSUE TITLE
Pipe Rifle crafting times inconsistent between calibers.

### ISSUE THREAD

**Issue by Cynthius** [CleverRaven/Cataclysm-DDA#70731]
### Describe the bug

The recipe for crafting a .22 pipe rifle takes half as long as the recipe for crafting every other caliber of pipe rifle except 9x19mm.

Likewise, the 9x19mm pipe rifle takes 3x longer than any other pipe rifle to craft. 6x longer compared to the .22.

### Attach save file

N/A

Would've provided a save file, but the game crashed with an error message when I tried to generate the save archive.

### Steps to reproduce

1. Load a character with the crafting recipes for Pipe Rifles unlocked. 3 Fabrication, 1 Marksmanship, 1 Devices.
2. Open the crafting menu and check the Weapons -> Ranged tab
3. Expand the pipe rifles category and compare crafting times between each variant

### Expected behavior

All pipe rifle recipes having the same (or similar) crafting times. 

All of the recipes are otherwise identical, and use the same components and tools; a metal saw, a screwdriver, a plank, a pipe, and 2 scrap metal, so I'm assuming this disparity between these two calibers and the rest is probably unintended. 

### Screenshots

![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/15305144/4dd2a00d-d161-4613-9f5d-9c68c6caab11)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/15305144/ceb25900-d582-4ae0-95cb-1922ba86f222)
![image](https://github.com/CleverRaven/Cataclysm-DDA/assets/15305144/dd52ca35-de3d-4d5b-bb76-0995b52d5e63)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3803 (22H2)
- Game Version: 7c80ee9 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Bionic Professions [package_bionic_professions],
    sees-player, run, crouch, bleed icons, retrodays [sees_player_retro],
    Stats Through Skills [StatsThroughSkills]
]

**Comment by RenechCDDA** [comment]
>Would've provided a save file, but the game crashed with an error message when I tried to generate the save archive.

Please provide this. It's not related to your report, but we still want to see it.

**Comment by Cynthius** [comment]
The save file, or the error message? I've tried generating the archive again, and the game crashed with the same error.

**Comment by DragonWizard23** [comment]
I can confirm that the 9x19mm pipe rifle takes way longer and the .22 pipe rifle is quicker. 
![Screenshot (284)](https://github.com/CleverRaven/Cataclysm-DDA/assets/151102088/8780e0b0-a08a-48ab-815a-453aad9aa90d)
![Screenshot (285)](https://github.com/CleverRaven/Cataclysm-DDA/assets/151102088/fc34d306-a948-4a93-8044-29954e0eec05)
Funny enough, the even 9x18mm pipe rifle is faster to craft than the 9x19mm one. 
![Screenshot (286)](https://github.com/CleverRaven/Cataclysm-DDA/assets/151102088/054bf309-5e09-4b19-b1c8-11a5c4650f03)
![Screenshot (287)](https://github.com/CleverRaven/Cataclysm-DDA/assets/151102088/ee3fa44b-2ef5-48b1-987b-e1d05374ee88)

**Comment by RenechCDDA** [comment]
> The save file, or the error message? I've tried generating the archive again, and the game crashed with the same error.

Yes to both. debug.log and crash.log in the /config/ folder if they exist for the error message

**Comment by RenechCDDA** [comment]
@andrei8l Reporting a crash on trying to export a save archive, figure you want to be aware. (Can move that part to a separate issue if needed)

**Comment by andrei8l** [comment]
>TYPE: class std::filesystem::filesystem_error
MESSAGE: symlink_status: The parameter is incorrect.: "./save/Scarcity\worldoptions.json"

This looks like an upsteam issue for non-NTFS filesystems. Are you running CDDA from an (ex)FAT partition (ex: external drive), or a cloud backup, or directly from the .zip? If yes just put an emoji here, otherwise please open a separate issue with that first crash log.

**Comment by RenechCDDA** [comment]
Back on topic: The pipe rifles do indeed have different crafting times, and the difference does seem to be pretty arbitrary. These recipes could use a bit of work (why are some of them *reversible*?!), so I'm going to mark it confirmed.

Here's a basic outline of what can be done to resolve the issue:

-Make the crafting time the same for all of them

-Make sure none of these recipes are reversible (you can't un-bore a pipe, or un-carve some wood furniture)
--Add carving to the proficiencies used, but at a very low modifier (it is only a portion of the time)

-(SEMI-OPTIONAL) Some sort of better representation of the striking hammer would be good but someone more familiar with firearm construction would know what works. Two bits of scrap metal floating in the air should not be able to fire a cartridge! You probably need e.g. a nail held under tension (a spring?)

-(OPTIONAL) Make a requirements_set for the shared requirements of pipe, plank, and 2 scrap metal so that these recipes can be easily modified in the future.
